test/export/test_export.py

```diff
@@ -6208,6 +6205,7 @@ def forward(self, x: Input):
                     else:
                         self.assertIsInstance(s, int)
 
+    @testing.expectedFailureRetraceability  # T186979579
```

Comment:
why is this failing now?

Reply:
lolwut, this should have been on the test above! merge weirdness